docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -199,12 +199,14 @@ For example, you have to set `image` to `your.private.registry.com/deckhouse:v1.
 To switch the Deckhouse cluster to using a third-party registry, follow these steps:
 
 * Update the `image` field in the `d8-system/deckhouse` deployment to contain the address of the Deckhouse image in the third-party-registry;
-* Edit the `d8-system/deckhouse-registry` secret (note that all parameters are Base64-encoded):
-  * Insert third-party registry credentials into `.dockerconfigjson`.
-  * Replace `address` with the third-party registry's host address (e.g., `registry.example.com`).
-  * Change `path` to point to a repo in the third-party registry (e.g., `/deckhouse/ee`).
-  * If necessary, change `scheme` to `http` (if the third-party registry uses HTTP scheme).
-  * If necessary, change or add the `ca` field with the root CA certificate that validates the third-party registry's https certificate (if the third-party registry uses self-signed certificates).
+* Download script [change-registry.sh](https://github.com/deckhouse/deckhouse/blob/main/tools/change-registry.sh) to the master node and start them with third-party-registry parameters.
+  * Example:
+
+    ```shell
```

Comment:
I suggest adding a command to download script like [here](https://github.com/deckhouse/deckhouse/blob/main/docs/documentation/pages/DECKHOUSE-FAQ.md?plain=1#L164).

Reply:
yep. added